clients/fides-js/src/lib/consent-link.tsx

```diff
@@ -0,0 +1,54 @@
+import { h, Component, VNode } from 'preact';
+import {CookieKeyConsent, hasSavedConsentCookie} from "./cookie";
+import debugLog, {getBannerOptions} from "~/lib/consent-utils";
+import {ConsentBannerOptions} from "~/lib/consent-types";
+
+
+interface LinkProps {
+    defaults: CookieKeyConsent;
+    options: ConsentBannerOptions;
+}
+
+class ConsentLink extends Component<LinkProps> {
+
+    private readonly options: ConsentBannerOptions;
+
+    constructor(props: LinkProps) {
+        super(props);
+        this.options = props.options;
+    }
+
+
+    /**
+     * Builds the DOM elements for the consent banner (container, buttons, etc.) and
+     * return a single div that can be added to the body.
+     */
+    buildLink = () => {
+        const options: ConsentBannerOptions = getBannerOptions();
+        // todo- implement opening modal depending on privacy experience
+        return (
+            <a href={options.privacyCenterUrl}>{options.consentLinkText}</a>
+            );
+    };
+
+    render(): VNode | null {
+        let linkBuild = null
+        // If the user provides any extra options, override the defaults
+        try {
+            if (hasSavedConsentCookie()) {
+                // todo- check for device id too
+                // also does the existence of a device id prevent links from rendering too, or just banners?
+                debugLog("Fides consent cookie already exists, skipping banner initialization!");
+                return null;
+            }
```

Comment:
Why would the link rendering depend on the cookie?

Reply:
I just didn't see this acceptance criteria explicitly addressed. I'll clarify with @mfbrown

Reply:
Apologies for the confusion here. The link should always be present in the locations for which it is configured.